bouncerscript/src/bouncerscript/constants.py

```diff
@@ -7,6 +7,14 @@
     "thunderbird-latest-ssl": r"^Thunderbird-\d+\.\d+(\.\d+)?-SSL$",
     "thunderbird-msi-latest-ssl": r"^Thunderbird-\d+\.\d+(\.\d+)?-msi-SSL$",
     "thunderbird-msix-latest-ssl": r"^Thunderbird-\d+\.\d+(\.\d+)?-msix-SSL$",
+    "thunderbird-esr-latest": r"^Thunderbird-\d+\.\d+(\.\d+)?esr$",
```

Comment:
Are non-ssl aliases actually needed?

Reply:
non-SSL releases refer to the ancient days when the US couldn't export encryption tech, correct?

Probably not technically needed... thunderbird.net does not refer to them.

Reply:
I'm not sure TBH, it predates my involvement by some years, I assume it's something like that, a relic from the days where https wasn't ubiquitous, which feels obsolete.  I'm happy to keep this here for now if you want though, it doesn't really hurt anything, and the client code can decide which aliases to actually populate/update, let me know.